client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/utils/style-variations.ts

```diff
@@ -14,3 +15,18 @@ export function removeLegacyDesignVariations( designs: Design[] ) {
 		);
 	return designs.filter( ( design ) => ! isVariationOfAnotherDesign( design ) );
 }
+
+/**
+ * Promote designs with style variations by having them at the beginning of the list.
+ */
+export function promoteDesignVariations( designs: Design[] ) {
+	const designsWithVariations = designs.filter(
+		( design ) => design.style_variations && design.style_variations.length > 0
+	);
+
+	const designsWithoutVariations = designs.filter(
+		( design ) => design.style_variations && design.style_variations.length === 0
+	);
+
+	return shuffle( designsWithVariations ).concat( shuffle( designsWithoutVariations ) );
```

Comment:
I think we don't need to shuffle again as we just re-order the list by `style_variations`

BTW, we could just use `designs.sort` to re-order them. What do you think?

Reply:
Correct, we don't need to shuffle again. We just need to move the them to the front, retaining the original order.

@arthur791004 I don't understand about `designs.sort`. We just need to concat the arrays, or am I missing something?

Reply:
Oh right... what was I thinking 🤦🏻‍♂️

Reply:
I think @arthur791004 was suggesting that instead of using two `filter` calls, we use one `sort` call for re-ordering the theme list. I've updated the code, let me know what you guys think 🙂